lmorpho/lsys_models.cpp

```diff
@@ -0,0 +1,139 @@
+#include <math.hpp>
+
+#include "lsystem.hpp"
+#include "lsys_models.hpp"
+
+static constexpr double inf = nest::mc::math::infinity<double>();
+
+// Predefined parameters for two classes of neurons. Numbers taken primarily
+// from Ascoli et al. 2001, but some details (soma diameters for example)
+// taken from earlier literature.
+//
+// Without mixture distribution support, these aren't entirely faithful to
+// the original sources.
+//
+// References:
+//
+// Ascoli, G et al. (2001),
+//     "Computer generation and quantitative morphometric analysis of virtual neurons."
+//      Anatomy and Embryology 204 (4) 283-301.
+//
+// Cullheim, S et al. (1987),
+//     "Membrane area and dendritic structure in type‐identified triceps surae alpha motoneurons."
+//     Journal of comparative neurology 255 (1) 68-81.
+//
+// Rapp, M et al. (1994).
+//     "Physiology, morphology and detailed passive models of guinea-pig cerebellar Purkinje cells."
+//     The Journal of Physiology, 474(1), 101-118.
+
+lsys_param make_alpha_motoneuron_lsys() {
+    lsys_param L;
+
+    // Soma diameter [µm].
+    L.diam_soma = { 47.0, 65.5, 57.6, 5.0 };
+
+    // Number of dendritic trees (rounded to nearest int).
+    L.n_tree = { 8.0, 16.0 };
+
+    // Initial dendrite diameter [µm]. (Dstem)
+    L.diam_initial = { 3.0, 12.0 };
+
+    // Dendrite step length [µm]. (ΔL)
+    L.length_step = { 25 };
+
+    // Initial roll (intrinsic rotation about x-axis) [degrees].
+    L.roll_initial = { -180.0, 180.0 };
```

Comment:
Yes, exactly. I've used degrees in the parameter sets to make it clearer to a reader, and to match the conventions used in the literature. This is also the reason why parameters that relate to a diameter have not been scaled to relate to the radii we use internally.